controller/workitem.go

```diff
@@ -66,6 +66,17 @@ func (c *WorkitemController) List(ctx *app.ListWorkitemContext) error {
 	if err != nil {
 		return jsonapi.JSONErrorResponse(ctx, errors.NewBadParameterError("could not parse filter", err))
 	}
+	if ctx.FilterExpression != nil {
+		q := *ctx.FilterExpression
+		// Better approach would be to convert string to Query instance itself.
+		// Then add new AND clause with spaceID as another child of input query
+		// Then convert new Query object into simple string
+		queryWithSpaceID := fmt.Sprintf(`{"$AND":[{"space": "%s" }, %s]}`, ctx.SpaceID, q)
```

Comment:
That approach will not redirect if a query object is not passed as expected. Hence blocking another request which will fail eventually (in a negative scenario).
But that approach will do conversion twice for the correct input query (in a happy scenario).